resolwe_bio/processes/support_processors/rnaseqc_qc.py

```diff
@@ -161,48 +180,66 @@ class Input:
-                description="Set the lower bound on read quality for exon coverage counting. Reads below this number are excluded from coverage metrics.",
+                description="Set the lower bound on read quality for exon coverage counting."
+                " Reads below this number are excluded from coverage metrics.",
             )
 
             base_mismatch = IntegerField(
                 label="Base mismatch [--base-mismatch]",
                 default=6,
-                description="Set the maximum number of allowed mismatches between a read and the reference sequence. Reads with more than this number of mismatches are excluded from coverage metrics",
+                description="Set the maximum number of allowed mismatches between a read and the reference sequence."
+                "Reads with more than this number of mismatches are excluded from coverage metrics",
             )
 
             offset = IntegerField(
-                label="Offset [ --offset]",
+                label="Offset [--offset]",
                 default=150,
-                description="Set the offset into the gene for the 3' and 5' windows in bias calculation. A positive value shifts the 3' and 5' windows towards eachother, while a negative value shifts them apart.",
+                description="Set the offset into the gene for the 3' and 5' windows in bias calculation."
+                "A positive value shifts the 3' and 5' windows towards each other, while a negative value shifts them apart.",
             )
 
             window_size = IntegerField(
                 label="Window size [--window-size]",
                 default=100,
-                description="Set the offset into the gene for the 3' and 5' windows in bias calculation. A positive value shifts the 3' and 5' windows towards eachother, while a negative value shifts them apart.",
+                description="Set the offset into the gene for the 3' and 5' windows in bias calculation."
+                "A positive value shifts the 3' and 5' windows towards eachother, while a negative value shifts them apart.",
             )
 
             gene_length = IntegerField(
                 label="Window size [--gene-length]",
                 default=600,
-                description="Set the minimum size of a gene for bias calculation. Genes below this size are ignored in the calculation.",
+                description="Set the minimum size of a gene for bias calculation."
+                "Genes below this size are ignored in the calculation.",
             )
 
             detection_threshold = IntegerField(
                 label="Detection threshold [--detection-threshold]",
                 default=5,
-                description="Number of counts on a gene to consider the gene 'detected'. Additionally, genes below this limit are excluded from 3' bias computation.",
+                description="Number of counts on a gene to consider the gene 'detected'."
```

Comment:
When you are breaking lines, adding a whitespace at the end is necessary. See for example [here](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/support_processors/multiqc.py#L405).